### PR TITLE
5-Arrays.fill vs Arrays.setAll methods 

### PR DESCRIPTION
**Pull request by ZytaWisz**
Showed examples how to using fill and setAll method in array and differences between them.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -73,6 +73,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${version.plugin.maven.compiler}</version>
+                <configuration>
+                    <source>14</source>
+                    <target>14</target>
+                </configuration>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:79]: If  in `<properties>` there is `<maven.compiler.release>` or `<maven.compiler.source>` & `<maven.compiler.target>` pair, than this config is redundant

File: pom.xml [hunk 2]
```diff
@@ -73,6 +73,10 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>${version.plugin.maven.compiler}</version>
+                <configuration>
+                    <source>14</source>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:77]: Downgrade from JDK 16

File: pom.xml [hunk 3]
```diff
@@ -73,6 +73,10 @@
...
+                    <target>14</target>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:78]: Downgrade from JDK 16

File: pom.xml [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.epam.prejap.ess</groupId>
     <artifactId>helpers</artifactId>
-    <version>0.1</version>
+    <version>0.3</version>
```
Comment (marcin-ogorzalek-EPAM) [inline pom.xml:9]: Good change

File: src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+
+    public static void main(String[] args) {
+
+        /**
+         * The fill method in an array allows to fill array in whole or fill from the start index to end index.
+         * The values must be the same type as the array.
+         * Examples below shown this two options to using fill methods.
+         */
+
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java:23]: one empty line too much

File: src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+/**
+ * Examples below shown differences between fill and setAll methods.
+ * <p>
+ * This methods comes from{@link Arrays} class.
+ * </p>
+ * @author Zyta Wiszniewska
+ * @version 0.3
+ */
+public class ComparisonFillAndSetAllMethods {
+
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java:14]: one empty line too much

File: src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java [hunk 7]
```diff
@@ -0,0 +1,65 @@
+package com.epam.prejap.ess.helpers.arrays;
+
+import java.util.Arrays;
+
+/**
+ * Examples below shown differences between fill and setAll methods.
```
Comment (marcin-ogorzalek-EPAM) [inline src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java:6]:
```suggestion
 * Examples below shown differences between {@link Arrays#fill(Object[], Object)} and {@link Arrays#setAll(Object[], IntFunction)} methods.
```
In cases like that use `{@link ...}` to provide a direct link to a given javadoc. Someone may use it to just jump there and read what is needed.

Comment (marcin-ogorzalek-EPAM) [reply src/main/java/com/epam/prejap/ess/helpers/arrays/ComparisonFillAndSetAllMethods.java:6]: To be more precise a direct link to the members of a class.

**Review by marcin-ogorzalek-EPAM — CHANGES_REQUESTED** [review]
Some changes are required and there is a merge conflict to resolve.

**Comment by ZytaWisz** [comment]
I corrected the code. Please review

**Review by marcin-ogorzalek-EPAM — APPROVED** [review]

**Pull request merged**